migrations/main.sql

```diff
@@ -650,3 +650,32 @@ CREATE VIEW user_request_booking_hostel_view AS
     ORDER BY
         u.university_id,
         s.user_id;
+
+
+DROP VIEW IF EXISTS user_document_list_view;
+CREATE VIEW user_document_list_view AS
+    SELECT
+        ur.university_id,
+        ud.created_at,
+        ud.updated_at,
+        ud.name,
+        ud.user_document_id,
+        ur.user_id
+    FROM
+        user_request ur
+	LEFT JOIN user_document ud ON
+		ur.service_id = ud.user_document_id
```

Comment:
Don't understand the connection between user_request.service_id and user_document.user_document_id

Reply:
May be you meant service_document_id?

Reply:
in document creation we use service_id. Without it we can't create document. So i want to show every document, that was created by service_id

Reply:
What values does ur.service_id can have? For now it 1 and 2. As to the ud.user_document_id it can be 1..... oo

Reply:
I've tested it and the result as I expected to be: the endpoint return first created documents with id's 1 and 2, for different student result the same, and the number of documents about 10.

Reply:
There is a need to implement different logic of this JOIN

Reply:
> What values does ur.service_id can have? For now it is 1 and 2 and it may be 3, 4, 5 in future. As to the ud.user_document_id it can be 1..... oo

you mean it 'dublicated'?